src/pathfinding_service/model/channel_view.py

```diff
@@ -42,25 +42,22 @@ def __post_init__(self) -> None:
         if self.capacity is None:
             self.capacity = self.deposit
 
-    # TODO: define another function update_deposit
+    def update_deposit(self, deposit: TokenAmount = TokenAmount(0)) -> None:
+
+        self.capacity = TokenAmount(self.capacity + deposit - self.deposit)
+        self.deposit = TokenAmount(self.deposit + deposit)
+
     def update_capacity(
```

Comment:
What happens if the PFS receives a update before it can get the `Deposit` event? I think we don't handle this correctly.